sllurp/llrp_proto.py

```diff
@@ -101,9 +102,11 @@
 ubyte_ushort_size = struct.calcsize('!BH')
 ubyte_uint_size = struct.calcsize('!BI')
 ushort_ubyte_size = struct.calcsize('!HB')
+ushort_ubyte_uint_size = struct.calcsize('!HBI')
 ushort_ushort_size = struct.calcsize('!HH')
 ushort_uint_size = struct.calcsize('!HI')
 uint_ubyte_size = struct.calcsize('!IB')
+uint_ubyte_ubyte_size = struct.calcsize('!IBB')
```

Comment:
I tried to order these lines by the number of "fields" in it.
So, the best for your added lines, would be to go a few lines later after ubyte_uint_uint.
I ordered by 1) number of fields, 2) fields size order.

Reply:
I have made another commit. Did not fully understand what you meant, by:

> I tried to order these lines by the number of "fields" in it.

Will fix in next commit.

Reply:
I was meaning, i order first by the number of "letters", and only as a second key by field size.
So, for example:
> ushort_ubyte_uint_size = struct.calcsize('!HBI')
should be at the end, after:
> ubyte_ushort_ushort_size = struct.calcsize('!BHH')
because there is 3 letters and that ushort_ is bigger than ubyte.